test/transform.js

```diff
@@ -60,6 +60,27 @@ export default {
 			],
 			expect: "foo.foo + foo.foo.foo",
 			description: "Use object literal spec"
+		},
+		{
+			args: [
+				"foo + bar * baz",
+				[
+					(node) => {
+						if (node.type === "BinaryExpression" && node.operator === "*") {
+							return {name: "prod", type: "Identifier"};
+						}
+					},
+					{
+						Identifier: (node) => {
+							if (node.name !== "foo") {
+								return {...node, name: "foo"};
+							}
+						}
+					},
+				]
+			],
+			expect: "foo + foo",
+			description: "Use array of callbacks"
```

Comment:
Interesting, I had never considered people may want to use `description` without `name`, just to get the default `name`. That's actually a nice validation of htest's sensible defaults!